plugins/Marketplace/vue/src/StartFreeTrial/StartFreeTrial.vue

```diff
@@ -5,103 +5,298 @@
+                @click="createAccountAndStartFreeTrial()"
+            >{{ translate('Marketplace_TrialStartNoLicenseCreateAccount' )}}</button>
+          </p>
+          <p v-html="$sanitize(trialStartNoLicenseAddHereText)" />
+        </div>
       </div>
-    </div>
+    </template>
   </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from 'vue';
 import {
   AjaxHelper,
+  externalLink,
   MatomoUrl,
   NotificationsStore,
   translate,
 } from 'CoreHome';
+import { Field } from 'CorePluginsAdmin';
+import Matomo from '../../../../CoreHome/vue/src/Matomo/Matomo';
 
 const { $ } = window;
 
+interface StartFreeTrialState {
+  createAccountEmail: string;
+  createAccountError: string | null;
+  trialStartError: string | null;
+  trialStartInProgress: boolean;
+  trialStartSuccessNotificationMessage: string;
+  trialStartSuccessNotificationTitle: string;
+  loadingModalCloseCallback: undefined | (() => void);
+}
+
 export default defineComponent({
+  components: { Field },
   props: {
     modelValue: {
       type: String,
       required: true,
     },
+    currentUserEmail: String,
     isValidConsumer: Boolean,
   },
+  data(): StartFreeTrialState {
+    return {
+      createAccountEmail: this.currentUserEmail || '',
+      createAccountError: null,
+      trialStartError: null,
+      loadingModalCloseCallback: undefined,
+      trialStartInProgress: false,
+      trialStartSuccessNotificationMessage: '',
+      trialStartSuccessNotificationTitle: '',
+    };
+  },
   emits: ['update:modelValue', 'trialStarted'],
   watch: {
     modelValue(newValue) {
-      if (newValue) {
-        if (this.isValidConsumer) {
-          this.startFreeTrial();
-        } else {
-          this.showLicenseDialog();
-        }
+      if (!newValue) {
+        return;
+      }
+
+      if (this.isValidConsumer) {
+        this.trialStartSuccessNotificationMessage = translate(
+          'CorePluginsAdmin_PluginFreeTrialStarted',
+          '<strong>',
+          '</strong>',
+          newValue,
+        );
+
+        this.startFreeTrial();
+      } else {
+        this.trialStartSuccessNotificationTitle = translate(
+          'CorePluginsAdmin_PluginFreeTrialStartedAccountCreatedTitle',
+        );
+
+        this.trialStartSuccessNotificationMessage = translate(
+          'CorePluginsAdmin_PluginFreeTrialStartedAccountCreatedMessage',
+          newValue,
+        );
+
+        this.showLicenseDialog(false);
       }
     },
   },
-  methods: {
-    linkTo(params: QueryParameters) {
-      return `?${MatomoUrl.stringify({
-        ...MatomoUrl.urlParsed.value,
-        ...params,
+  computed: {
+    trialStartNoLicenseAddHereText() {
+      const link = `?${MatomoUrl.stringify({
+        module: 'Marketplace',
+        action: 'manageLicenseKey',
       })}`;
+
+      return translate(
+        'Marketplace_TrialStartNoLicenseAddHere',
+        `<a href="${link}" target="_blank" rel="noreferrer noopener">`,
```

Comment:
It's safe to use `externalLink` here as well since the campaign params are only added if the domain is/ends with `matomo.org`, otherwise it returns the tag markup only for the given URL.

```suggestion
        externalLink(link),
```

Reply:
Pushed this myself.